source/geometries/Next100FieldCage.cc

```diff
@@ -404,12 +448,12 @@ void Next100FieldCage::BuildELRegion()
 {
   /// EL gap
   G4Tubs* el_gap_solid =
-    new G4Tubs("EL_GAP", 0., el_gap_diam_/2., el_gap_length_/2., 0, twopi);
+  new G4Tubs("EL_GAP", 0., el_gap_diam_/2., el_gap_length_/2., 0, twopi);
```

Comment:
In past geometries we used to place the grids inside the EL gap volume. Since they're very thin, it's irrelevant, as you say. I would keep it the same as the other geometries.